wiki/Help_centre/Tournament_bans/en.md

```diff
@@ -0,0 +1,54 @@
+# Tournament bans
+
+*Main page: [Help centre](/wiki/Help_centre)*
+
+A tournament ban prevents a player from being involved in officially supported tournaments in any capacity. This includes playing and staffing or any other kind of assistance.
+
+Tournament bans are very rarely handed out their own, but generally speaking almost all players who return under a standard appeal for an [account restriction](/wiki/Help_centre/Account_restrictions) are subject to one of at least a year in length.
```

Comment:
```suggestion
Tournament bans are very rarely handed out on their own, but generally speaking almost all players who return under a standard appeal for an [account restriction](/wiki/Help_centre/Account_restrictions) are subject to one of at least a year in length.
```

Maybe I'm being stupid, but the current wording feels wrong to me. Use your own judgment I guess.